examples/ai-core/src/lib/save-audio.ts

```diff
@@ -0,0 +1,30 @@
+import { GeneratedFile } from 'ai';
+import fs from 'node:fs';
+import path from 'node:path';
+
+const OUTPUT_DIR = 'output';
+const audioFormatMap = {
+  'audio/mpeg': 'mp3',
+  'audio/wav': 'wav',
+  'audio/flac': 'flac',
+  'audio/aac': 'aac',
+  'audio/ogg': 'ogg',
+};
```

Comment:
makes me wonder if we should also return the format in addition to the mime type

Reply:
but might be tricky given the relationship to GeneratedFile

Reply:
We could augment GeneratedFile later with a mime library or something, would be handy but those libraries are quite heavy.

Reply:
From convo, building `GeneratedAudioFile` where the format interpreter works for the class, so it's very easy to swap out for improved `GeneratedFile` later.